### ISSUE TITLE
Unwanted speakers showing while not enabled

### ISSUE THREAD

**Issue by mhoogenbosch** [johanfrick/custom-sonos-card#256]
Using latest version of both HA core and Sonos-card (respectively 2023.12.3 and 6.12.4)

With following code on my Windows 11 device, it shows only the four listed speakers. Which is what I'd like it to have.

```
type: custom:sonos-card
sections:
  - grouping
mediaBrowserItemsPerRow: 4
widthPercentage: 100
showVolumeUpAndDownButtons: false
entities:
  - media_player.eetkamer
  - media_player.woonkamer
  - media_player.veranda
  - media_player.slaapkamer
predefinedGroups:
  - name: Woon + Eet
    media: ''
    entities:
      - player: media_player.woonkamer
        volume: 12
      - player: media_player.eetkamer
        volume: 12
  - name: Woon + Eet + Veranda
    media: ''
    entities:
      - player: media_player.eetkamer
        volume: 12  
```

On my wallboard tablet (nTab plus on Android 12) and my iPhone 15 on iOS 17.2.1 it shows the SqueezeLite players too.

What i'd expect:

![image](https://github.com/johanfrick/custom-sonos-card/assets/36899598/01f701fc-fc1f-41df-adf7-8ad1dfd26c91)

On iphone, and wouldn't expect:

![Afbeelding van WhatsApp op 2023-12-27 om 12 01 03_d58447a0](https://github.com/johanfrick/custom-sonos-card/assets/36899598/6da898e0-dfc4-40cb-b0fb-b0e43bf587ea)

**Comment by mhoogenbosch** [comment]
Adding excludeItemsInEntitiesList: false solves it. Sorry

**Comment by johanfrick** [comment]
> Adding excludeItemsInEntitiesList: false solves it. Sorry

@mhoogenbosch That bug has been fixed now with [v6.15.3](https://github.com/johanfrick/custom-sonos-card/releases/tag/v6.15.3). It now defaults to false.
